### PR TITLE
Upgrade spring data to 2.2.2 & boot 2.5.1

### PR DESCRIPTION
**Pull request by IAM20**
Upgrade spring data version 2.2.2
Boot 2.5.1

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.5.0")
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.5.1")
```
Comment (mhyeon-lee) [inline build.gradle:12]: 2.5.2 로 가시죠

File: guide-projects/plus-repository-guide/build.gradle [hunk 2]
```diff
@@ -5,8 +5,8 @@ dependencies {
     implementation("org.projectlombok:lombok")
     annotationProcessor("org.projectlombok:lombok")
 
-    implementation("org.springframework.data:spring-data-jdbc:2.2.1")
-    implementation("org.springframework.data:spring-data-relational:2.2.1")
+    implementation("org.springframework.data:spring-data-jdbc:2.2.2")
+    implementation("org.springframework.data:spring-data-relational:2.2.2")
     implementation("org.springframework.data:spring-data-commons:2.5.1")
```
Comment (mhyeon-lee) [inline guide-projects/plus-repository-guide/build.gradle:10]: spring data commons 도 2.5.2

File: build.gradle (hunk 1 above)
Comment (IAM20) [reply build.gradle]: Reflected in b577ab0 :)

File: guide-projects/plus-repository-guide/build.gradle (hunk 2 above)
Comment (IAM20) [reply guide-projects/plus-repository-guide/build.gradle]: Reflected in b577ab0 :)
감사합니다..!

**Review by mhyeon-lee — APPROVED** [review]

**Pull request merged**